README.md

```diff
@@ -1,3 +1,10 @@
+## ⚠️ Deprecated!
+
+**This repository is deprecated.** <br />
+A more up-to-date version of examples is available here: https://github.com/grafana/grafana-plugin-examples
```

Comment:
Should we migrate this starter-panel into the examples or point it to the basic-panel?

Reply:
That's a good question. I pointed it to the "root" as we have multiple panel related examples - but pointing it to the basic panel could be a fair choice as well. What do you think?

Reply:
So what's the decision here? @jackw @leventebalogh